.github/workflows/ci.yml

```diff
@@ -5,9 +5,6 @@ on:
     branches:
       - main
   pull_request:
-  schedule:
-    # Run the workflow every Monday at 6 am CET
-    - cron: '0 4 * * MON'
```

Comment:
I didn't get to maintain the repo regularly this year and noticed that GitHub would turn off the scheduled CI after a certain period of inactivity.

For simplicity, I'll just let CI run for pull requests and merges only.